aidial_adapter_openai/app.py

```diff
@@ -100,30 +68,40 @@ def get_api_version(request: Request):
 
 @app.post("/openai/deployments/{deployment_id}/chat/completions")
 async def chat_completion(deployment_id: str, request: Request):
+
     data = await parse_body(request)
 
     is_stream = data.get("stream", False)
 
-    api_type, api_key = await get_credentials(request, chat_completions_parser)
+    if deployment_id in LEGACY_COMPLETIONS_DEPLOYMENTS:
```

Comment:
You really don't need to introduce `LEGACY_COMPLETIONS_DEPLOYMENTS` env var, because the format of the upstream endpoint already tells you if this is a chat completions model of a legacy completions model. The parsers do the work of distinguishing one from another.

Reply:
fixed